examples/privileged-pod-pvc/README.md

```diff
@@ -0,0 +1,83 @@
+# Mount Volumes on Privileged Pods
+
+**This guide will demo GlusterFS as its example use-case but this method will work for any compatable volume provider.**
+
+##Purpose
+
+This example gives a basic template for attaching a persistent storage volume to a pod. It provides an end to end setup that begins with the _cluster-admin_ making the persistent volume availble and a _basic-user_ requesting storage from a **privileged** pod.
+
+_If the pod is not run as privileged, skip Edit Privileged SCC section_
```

Comment:
maybe highlight the "Edit Privleged SCC" section to stand out better from the rest of the italic-d sentence?